packages/normalize/css/index.css

```diff
@@ -17,7 +17,7 @@ html {
   font-size: 0.875em;
   font-weight: var(--psTypeFontWeightRegular);
   line-height: 1.71429; /* calc(var(--psTypeLineHeightStandard) / var(--psTypeFontSizeSmall)); */
-  font-feature-settings: 'tnum' on, 'lnum' on;
+  font-feature-settings: 'tnum' on, 'lnum' on, 'liga' off;
```

Comment:
This seems to be the only thing I can find that's a new style.  Correct?

Even if that's the case, I don't see this influencing the data-css-hashes, because normalize is a peerDependency that's just expected to be on the page, added to the style cascade in the browser, not present at build time.